extensions/common/iam/identity-trust/identity-trust-service/src/main/java/org/eclipse/edc/iam/identitytrust/verification/MultiFormatPresentationVerifier.java

```diff
@@ -14,19 +14,27 @@
 
 package org.eclipse.edc.iam.identitytrust.verification;
 
-import org.eclipse.edc.identitytrust.model.CredentialFormat;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.eclipse.edc.identitytrust.model.VerifiablePresentationContainer;
+import org.eclipse.edc.identitytrust.verification.JwtVerifier;
 import org.eclipse.edc.identitytrust.verification.PresentationVerifier;
 import org.eclipse.edc.spi.result.Result;
 
 public class MultiFormatPresentationVerifier implements PresentationVerifier {
-    private final JwtPresentationVerifier jwtVerifier = new JwtPresentationVerifier();
-    private final JsonLdPresentationVerifier jsonLdVerifier = new JsonLdPresentationVerifier();
+    private final JwtPresentationVerifier jwtPresentationVerifier;
+    private final JsonLdPresentationVerifier jsonLdVerifier;
```

Comment:
```suggestion
    private final JsonLdPresentationVerifier jsonLdPresentationVerifier;
```

to be consistent with the `JwtPresentationVerifier`